mutable-utils/src/test/kotlin/at/kopyk/TestMutation.kt

```diff
@@ -0,0 +1,14 @@
+package at.kopyk
+
+import io.kotest.matchers.types.shouldBeSameInstanceAs
+
+internal fun <T> testMutation(
```

Comment:
Loving this combinator!

Reply:
Thank you 🥰
I love to create things like these as well as test scopes for more complex cases. Beats doing shared setup and teardown strategies in so many ways hehe.